R/st_nyc_housing_prices.R

```diff
@@ -0,0 +1,3 @@
+saratoga_houses <- read.csv("https://wjhopper.github.io/SDS-201/data/saratoga_ny_home_prices.csv")
```

Comment:
`R/st_nyc_housing_prices.rda` should not be part of the PR. Please delete it and commit the deletion.